src/pages/Map.vue

```diff
@@ -15,8 +15,9 @@ export default {
   components: { GroupMap },
   computed: {
     ...mapGetters({
-      stores: 'stores/byActiveGroup',
-      users: 'users/all',
+      stores: 'stores/withLocation',
```

Comment:
Is this really meaning `byActiveGroupWithLocation`?

from reading just the name it looks like it could overfetch the stores, and show ones for other groups.